src/openforms/submissions/tests/form_logic/test_fetching_form_variable_values_from_services.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 from typing import Any
 from unittest import skip
-from urllib.parse import unquote_plus
+from urllib.parse import unquote
```

Comment:
unquote_plus in the documentation has remarks about it being for form data, not intended for URL paths. The behaviour is also slightly changed with us using `furl` for the final URL building instead of `urljoin`, and furl already encodes a couple of things